File: elixir/codewars/fizz_buzz.ex
# https://www.codewars.com/kata/fizz-buzz

defmodule FizzBuzz do
  def fizzbuzz(n) do
    1..n
    |> Enum.map(fn
      x when rem(x, 15) == 0 -> "FizzBuzz"
      x when rem(x, 3)  == 0 -> "Fizz"
      x when rem(x, 5)  == 0 -> "Buzz"
      x -> x
      end)
  end
end

File: py/codewars/fizz_buzz.py
#!/usr/bin/env python3
# https://www.codewars.com/kata/fizz-buzz

def fizzbuzz(n):
    return [
        'FizzBuzz' if x % 15 == 0 else
        'Fizz'     if x % 3  == 0 else
        'Buzz'     if x % 5  == 0 else
        x for x in range(1, n + 1)]
